client/client.go

```diff
@@ -21,7 +21,7 @@
 // propagated down the pipeline, with the values being produced by
 // authenticators and/or receivers, and consumed by processors and exporters.
 //
-// Producers
+// # Producers
```

Comment:
Are changes in this file relevant?

Reply:
This was all done automatically by running `make fmt`, it was causing lint to fail previously.